kernel_tuner/core.py

```diff
@@ -367,7 +369,7 @@ def benchmark_default(self, func, gpu_args, threads, grid, result, flush_cache=T
         ]
 
         self.dev.synchronize()
-        for _ in range(self.iterations):
+        for i in range(self.iterations):
```

Comment:
`i` doesn't seem to be used below. the for-loop on line 377 even defines its own `i`.